clientapp/src/components/Card.jsx

```diff
@@ -150,14 +153,19 @@ export class Card extends React.Component{
     getCardDates() {
```

Comment:
I think this would be more clear as two separate functions: getCardDates and getCardTimes